cumulus/zombienet/zombienet-sdk/tests/sync_blocks/sync_blocks_from_tip_without_connected_collator.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+
+use cumulus_zombienet_sdk_helpers::assert_para_throughput;
+use polkadot_primitives::Id as ParaId;
+use subxt::{OnlineClient, PolkadotConfig};
+use zombienet_sdk::{AddCollatorOptions, LocalFileSystem, Network, NetworkConfigBuilder};
+
+const PARA_ID: u32 = 2000;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn sync_blocks_from_tip_without_connected_collator() -> Result<(), anyhow::Error> {
+	let _ = env_logger::try_init_from_env(
+		env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+	);
+
+	let network = initialize_network().await?;
+
+	let relay_alice = network.get_node("alice")?;
+
+	let relay_client: OnlineClient<PolkadotConfig> = relay_alice.wait_client().await?;
+
+	// Ensure parachains are registered.
+	assert_para_throughput(&relay_client, 2, [(ParaId::from(PARA_ID), 2..3)].into_iter().collect())
+		.await?;
+
+	// Ensure parachains made progress.
+	assert_para_throughput(
+		&relay_client,
+		10,
+		[(ParaId::from(PARA_ID), 9..11)].into_iter().collect(),
+	)
+	.await?;
+
+	let para_ferdie = network.get_node("ferdie")?;
+	let para_eve = network.get_node("eve")?;
+
+	// Ensure ferdie and eve are syncing
+	let block_height_metric = "block_height{status=\"best\"}";
+	let timeout_secs: u64 = 250;
+	let min_block_height: f64 = 12.0;
+
+	assert!(para_ferdie
+		.wait_metric_with_timeout(block_height_metric, |b| b > min_block_height, timeout_secs)
```

Comment:
In the [issue description](https://github.com/paritytech/polkadot-sdk/issues/8129) it was suggested to use subxt.
I am querying metrics here to get the block height. Such approach allows to create more similar assertion to the one used in `zombienet`-based implementation:
```
ferdie: reports block height is at least 12 within 250 seconds
```
Just want to make sure if this is acceptable. If not then I am happy to refactor it :)

Reply:
Sounds good! Thanks @skunert :)